app/views/calculators/leave_pot_untouched/_estimate.html.erb

```diff
@@ -21,11 +21,14 @@
 
 <ul class="t-calculator-notes">
   <li>
-    This is an estimate based on your pot growing at a rate of about
+    This is an estimate based on your whole pot growing at a rate of about
     <%= number_to_percentage(LeavePotUntouchedCalculator::INTEREST_RATE * 100) %>
     per year — this may vary.
   </li>
   <li>
     The amount in your pot will be affected by inflation and any fees your provider charges.
   </li>
+  <li>
+  You must leave your whole pot – if you take a 25% tax-free lump sum, you must take the rest of your money within 6 months, using one of the [other pension options](/pension-pot-options)
```

Comment:
That's done now

On 10 March 2016 at 10:45, Nicola Albon <nicola.albon@pensionwise.gov.uk>
wrote:

> No, it should have a full stop. I'll go in and add that.
>
>
> On 10 March 2016 at 10:40, Andrew Garner <notifications@github.com> wrote:
>
>> In app/views/calculators/leave_pot_untouched/_estimate.html.erb
>> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/489#discussion_r55661376>
>> :
>>
>> >      <%= number_to_percentage(LeavePotUntouchedCalculator::INTEREST_RATE * 100) %>
>> >      per year — this may vary.
>> >    </li>
>> >    <li>
>> >      The amount in your pot will be affected by inflation and any fees your provider charges.
>> >    </li>
>> > +  <li>
>> > +  You must leave your whole pot – if you take a 25% tax-free lump sum, you must take the rest of your money within 6 months, using one of the [other pension options](/pension-pot-options)
>>
>> The previous two points end with a full stop but this one doesn't?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/489/files#r55661376>
>> .
>>
>
>